packages/laconia-core/test/laconia.spec.js

```diff
@@ -171,6 +171,69 @@ describe("laconia", () => {
     });
   });
 
+  describe("#set", () => {
+    describe("when registering a factory", () => {
+      it("should be able to add instances by calling 'set'", async () => {
+        const app = jest.fn();
+        await laconia(app)
+          .set("foo", lc => "bar")
+          .set("boo", lc => "baz")(...handlerArgs);
+
+        expect(app).toBeCalledWith(
+          expect.any(Object),
+          expect.objectContaining({
+            foo: "bar",
+            boo: "baz"
+          })
+        );
+      });
+
+      it("should be able to add async instances by calling 'set'", async () => {
+        const app = jest.fn();
+        await laconia(app).set("foo", async lc => {
+          return Promise.resolve("bar");
```

Comment:
You don't need to use `Promise.resolve` when returning from an async function, fyi.